src/science/casa-cnp/casa_cnp.F90

```diff
@@ -1341,13 +1341,11 @@ SUBROUTINE casa_delsoil(veg,casapool,casaflux,casamet,casabiome)
                   +casaflux%Psimm(nland)
              ! net mineralization
 
-#           ifdef ESM15  
-             casaflux%Pleach(nland)  =  (1.0e-4) &
-                                              * max(0.0,casapool%Psoillab(nland))
-#           else
+             !rml 14/10/24 #278 remove ESM15 specific version as can be 
+             !accommodated by setting appropriate parameter in pftlookup
```

Comment:
Instead of providing a comment here, is there any documentation on pftlookup versioning (similar to [bench_example](https://github.com/CABLE-LSM/bench_example/tree/main/namelists) repo?

Reply:
I'm not aware of any documentation on pftlookup versioning. It might be good to add a comment to the appropriate row of the table in this page of the userguide, https://cable.readthedocs.io/en/latest/user_guide/inputs/pftlookup_csv/ to note that xfPleach was hard-wired in CABLE2.4 for ACCESS-ESM1.5. I haven't made changes to the documentation yet so will need to work out how to do that.

Reply:
I've now made the change to the documentation that I suggested. I'd prefer to also keep the comment in the code as that allows anyone to refer back to the issue.